src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+			</li>
+			<li>
+				<a href="/sdk/software/api/reference/#ConditionType">ConditionType</a> - The <code className="small-code-sample">NoDVIRCheck</code> <code className="small-code-sample">ConditionType</code> is obsolete and will be removed in a future version. Please use NoPreDVIRCheck and NoPostDVIRCheck.
+			</li>
+		</ul>
+		<h2>Zones</h2>
+		<ul>
+			<li>
+				<a href="/sdk/software/api/reference/#Zone">Zone</a> - Added the <code className="small-code-sample">ZoneTypes</code> property for enumeration of zone types for a given zone.
+			</li>
+		</ul>
+		<h2>Devices</h2>
+		<ul>
+			<li>
+				<a href="/sdk/software/api/reference/#Go9">GO9</a> - Added the <code className="small-code-sample">ObdAlertEnabled</code> property to allow users to enable/disable OBD alerts on their vehicles.
+			</li>
+			<li>
+				<a href="/sdk/software/api/reference/#GoDevice">GoDevice</a> - Added the <code className="small-code-sample">ParameterVersionOnDevice</code> property to track the current parameter version on the device. The current <code className="small-code-sample">ParameterVersion</code> property communicates the parameter version to the device; however, parameter updates are not always immediate.
+			</li>
+			<li>
+				<a href="/sdk/software/api/reference/#Device">Device</a> - To prevent mismatches based on system clock settings, we have prevented <code className="small-code-sample">ActiveFrom</code> from being greater than <code className="small-code-sample">ActiveTo</code> when adding a device.
+			</li>
+		</ul>
+		<h2>Generator-Addin Version 3.0</h2>
+		<p>We have modernized the Add-In scaffolding, development and packaging tool to use more current techniques and features:</p>
+		<ul>
+			<li>
+				Now using webpack.
+			</li>
+			<li>
+				Now using Puppeteer for browser testing.
+			</li>
+			<li>
+				UI now shows a collapsible navbar.
+			</li>
+			<li>
+				Can now toggle multi-language support.
+			</li>
+			<li>
+				Can now toggle blur and focus events to simulate leaving and re-visiting the Add-In page.
+			</li>
+			<li>
+				For more information visit GitHub: <a href="https://github.com/Geotab/generator-addin" target="_blank">https://github.com/Geotab/generator-addin</a>
+			</li>
+		</ul>
+		<h2>Mg-Api-Js Version 2.0</h2>
+		<p>This major release merges the API wrappers mg-api-js (previously browser only version) and mg-api-node (previously Nodejs only version) into a single project:</p>
+		<ul>
+			<li>
+				Uses single js library for nodejs or browser.
+			</li>
+			<li>
+				Supports Async promises and legacy callback behavior.
+			</li>
+			<li>
+				Simplifies authentication process, no more hard-to-understand callbacks.
+			</li>
+			<li>
+				Optional lower-level control over http response.
+			</li>
+			<li>
+				For more information visit GitHub: <a href="https://github.com/Geotab/mg-api-js" target="_blank">https://github.com/Geotab/mg-api-js</a>
```

Comment:
<a> tags that point outside of the site should have target="_blank" rel="noopener noreferrer" attributes